### ISSUE TITLE
oclint - How can I define `FL_OCLINT_EXCLUDE_REGEX` value in a `.env` file

### ISSUE THREAD

**Issue by gtatgh** [fastlane/fastlane#10117]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

How can I define `FL_OCLINT_EXCLUDE_REGEX` value in a `.env` file? If try to use `FL_OCLINT_EXCLUDE_REGEX=/src/` then I get `type mismatch: String given`.

##### Complete output when running fastlane, including the stack trace and command used

```
DEBUG [2017-08-23 09:00:27.49]: Checking if there are any plugins that should be loaded...
INFO [2017-08-23 09:00:27.60]: gem 'spaceship' is already installed
INFO [2017-08-23 09:00:27.61]: -------------------------------------------------
INFO [2017-08-23 09:00:27.61]: --- Step: Verifying required fastlane version ---
INFO [2017-08-23 09:00:27.61]: -------------------------------------------------
INFO [2017-08-23 09:00:27.61]: Your fastlane version 2.54.0 matches the minimum requirement of 2.50.0  ✅
INFO [2017-08-23 09:00:27.61]: ------------------------------
INFO [2017-08-23 09:00:27.61]: --- Step: default_platform ---
INFO [2017-08-23 09:00:27.61]: ------------------------------
INFO [2017-08-23 09:00:27.62]: Driving the lane 'ios run_analyzer' 🚀
INFO [2017-08-23 09:00:27.62]: Lane run_analyzer started.
INFO [2017-08-23 09:00:27.62]: --------------------
INFO [2017-08-23 09:00:27.62]: --- Step: oclint ---
INFO [2017-08-23 09:00:27.62]: --------------------
DEBUG [2017-08-23 09:00:27.62]: Taking value for 'compile_commands' from environment variable 'FL_OCLINT_COMPILE_COMMANDS'
DEBUG [2017-08-23 09:00:27.62]: Taking value for 'compile_commands' from environment variable 'FL_OCLINT_COMPILE_COMMANDS'
DEBUG [2017-08-23 09:00:27.62]: Taking value for 'select_regex' from environment variable 'FL_OCLINT_SELECT_REGEX'
DEBUG [2017-08-23 09:00:27.62]: Taking value for 'select_regex' from environment variable 'FL_OCLINT_SELECT_REGEX'
DEBUG [2017-08-23 09:00:27.62]: Taking value for 'exclude_regex' from environment variable 'FL_OCLINT_EXCLUDE_REGEX'
INFO [2017-08-23 09:00:33.95]: Lane run_analyzer failed to complete.
WARN [2017-08-23 09:00:33.95]: Lane Context:
INFO [2017-08-23 09:00:33.95]: {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios run_analyzer"}
ERROR [2017-08-23 09:00:33.95]: type mismatch: String given
INFO [2017-08-23 09:00:33.96]: Successfully generated documentation at path '/Users/user/projects/ios/project/MyProject/fastlane/README.md'

+------+------------------------------+-------------+
|                 fastlane summary                  |
+------+------------------------------+-------------+
| Step | Action                       | Time (in s) |
+------+------------------------------+-------------+
| 1    | Verifying required fastlane  | 0           |
|      | version                      |             |
| 2    | default_platform             | 0           |
| 💥   | oclint                       | 0           |
+------+------------------------------+-------------+

ERROR [2017-08-23 09:00:33.96]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...
Search query: type mismatch: String given

URL: https://api.github.com/search/issues?q=type%20mismatch:%20String%20given+repo:fastlane/fastlane
➡️  oclint - How can I define `FL_OCLINT_SELECT_REGEX` value in a `.env` file
    https://github.com/fastlane/fastlane/issues/10069 [closed] 6 💬
    6 hours ago

🔗  You can ⌘ + double-click on links to open them directly in your browser.
bundler: failed to load command: fastlane (/Users/user/.gem/bin/fastlane)
TypeError: [!] type mismatch: String given
  /Users/user/3party/fastlane/fastlane/lib/fastlane/actions/oclint.rb:48:in `=~'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/actions/oclint.rb:48:in `!~'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/actions/oclint.rb:48:in `block in run'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/actions/oclint.rb:44:in `select!'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/actions/oclint.rb:44:in `run'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:226:in `chdir'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
  Fastfile:140:in `block (2 levels) in parsing_binding'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
  /Users/user/.gem/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
  /Users/user/.gem/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
  /Users/user/.gem/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
  /Users/user/3party/fastlane/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:64:in `run!'
  /Users/user/.gem/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
  /Users/user/3party/fastlane/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
  /Users/user/3party/fastlane/bin/fastlane:20:in `<top (required)>'
  /Users/user/.gem/bin/fastlane:23:in `load'
  /Users/user/.gem/bin/fastlane:23:in `<top (required)>'
```

**Comment by gtatgh** [comment]
Fixed with ba9ce3734ab106773c8d817dca38b9d207eafbe3

**Comment by gtatgh** [comment]
Fixed with ba9ce3734ab106773c8d817dca38b9d207eafbe3.
